src/components/tabs/Tabs.tsx

```diff
@@ -272,7 +272,7 @@ export const Tabs: FunctionComponent<TabsProps> = ({
           return (
             <React.Fragment key={key}>
               <TabsContent>{children}</TabsContent>
-              {wizard && (
+              {buttons != null && (
```

Comment:
Do we want to have a weak check against null here? Or we want just to check for truthy values?

Reply:
We're filtering out undefined & null values (they can come from optional props or be directly specified by user - ReactNode type accepts them), it's a common pattern across our codebase to use this check instead of strong 
`x !== undefined or !== null`